src/meshes/t8code_mesh.jl

```diff
@@ -1263,6 +1441,66 @@ function fill_mesh_info!(mesh::T8codeMesh, interfaces, mortars, boundaries,
     return nothing
 end
 
+function get_levels(mesh::T8codeMesh)
+    return trixi_t8_get_local_element_levels(mesh.forest)
+end
+
+function get_cmesh_info(mesh::T8codeMesh)
+    @assert t8_forest_is_committed(mesh.forest) != 0
```

Comment:
```suggestion
    @assert t8_forest_is_committed(mesh.forest)
```
Similar question to above.

Reply:
Same answer as above. :D

Reply:
If the only possible values are 0 and 1 could be turn this into
```suggestion
    @assert t8_forest_is_committed(mesh.forest) == 1
```
for more consistency and I think a bit better readability?

Reply:
Fair enough.